dashboard/test/dsl/script_dsl_test.rb

```diff
@@ -593,7 +593,11 @@ def setup
   test 'Script DSL for lesson with lesson group' do
     input_dsl = <<~DSL
       lesson_group 'required', display_name: 'Overview'
+      lesson_group_description 'This is a description'
+      lesson_group_question 'Question 1'
+      lesson_group_question 'Question 2'
```

Comment:
sad we're using DSL to store this. but agree this is the best path forward.

when we get to editing lessons, we shouldn't be adding anything new as DSL.